docs/configuration/OVERVIEW.md

```diff
@@ -36,103 +44,147 @@ A component will have to handle different types of configurations.
+  fields from the global configuration. (An example of this is available [below](#handle-global-config).)
+
+### Input configuration
 
-### Input config
+- The configuration of a __component instance__ can be overridden via `@Input` configuration.
 
-- The configuration of a __component instance__ can be overridden via @Input config
+### Priorities for the configuration
 
-__The priorities for the config__:
+- The highest priority is the one passed as input from a parent component
+- The second priority is the priority by component ID set dynamically
+- The lowest priority is the default configuration set in the component (in the `config.ts` file of the component)
 
-- the highest priority is the one passed as input from a parent component
-- the second priority is the priority by component type set in the store
-- the lowest priority is the default config set on component (in the config.ts file of the component)
+## How to set up the configuration service
 
-## Cases we handle
+We support two ways of setting up the configuration service (`ConfigurationBaseService`). It can be at the bootstrap of the
+application for a static configuration or at any time during application execution for a dynamic configuration. 
 
 ### Static configuration (override)
 
-- The default configuration is extracted by __cms-adapters__ tool and bundled in a JSON file
+- The default configuration is extracted by the __@o3r/components:extractor__ and bundled in a JSON file.
 
-- From this file we extract and modify the components configurations we want to customize, and we create a new JSON file.
+- From this file, we extract and modify the component's configurations we want to customize, and we create a new JSON file.
 
-- The content of the new file is injected in the index.html body's data attribute (_data-staticconfig_)
+- The content of the new file is injected in the `index.html` body's data attribute (`data-staticconfig`).
 
-- The value from the data attribute is taken and used to update the configuration store
+- The value from the data attribute is taken and passed to the configuration service.
 
-The big gain here is that the custom config is set up at the bootstrap of the application and available for the
-components at their instantiation time
+The big gain here is that the custom configuration is set up at the bootstrap of the application and available for the
+components when they are instantiated.
 
 ### Dynamic configuration
 
-- The default configuration is extracted by __cms-adapters__ tool and bundled in a JSON file will be exposed in the CMS
+- The default configuration is extracted by __@o3r/components:extractor__ and bundled in a JSON file which will be exposed in the CMS
 
-- In the CMS plugin the Business Analyst will modify configuration of components and this ones will be exposed (in the
+- In the CMS plugin the Business Analyst will modify the configuration of components which will be exposed (in the
```

Comment:
Do we really mention the CMS plugin when we do not provide it? 
I would be more generic "In a Configuration UI," or something like this